src/black/output.py

```diff
@@ -8,34 +8,40 @@
 from mypy_extensions import mypyc_attr
 import tempfile
 
-from click import echo, style
+from click import echo, style as _style
+
+from black.const import OutputLevels
 
 
 @mypyc_attr(patchable=True)
 def _out(message: Optional[str] = None, nl: bool = True, **styles: Any) -> None:
     if message is not None:
-        if "bold" not in styles:
-            styles["bold"] = True
-        message = style(message, **styles)
+        message = _style(message, **styles)
     echo(message, nl=nl, err=True)
 
 
 @mypyc_attr(patchable=True)
-def _err(message: Optional[str] = None, nl: bool = True, **styles: Any) -> None:
-    if message is not None:
-        if "fg" not in styles:
-            styles["fg"] = "red"
-        message = style(message, **styles)
-    echo(message, nl=nl, err=True)
+def out(
+    message: Optional[str] = None,
+    nl: bool = True,
+    style: OutputLevels = OutputLevels.null,
+    **_styles: Any,
+) -> None:
+    styles = style.value.copy()
+    if style:
```

Comment:
To me this is redundant because `.update` with an empty dict does nothing. Also discussed before.

Reply:
Yeah, but I wanted to keep the `.update()` change explicit hence keeping it in an `if` block.

Reply:
This seems wrong, doesn't it mean we ignore `_styles` if `style == LogLevel.none`?

Also, the variable names `style`, `styles`, and `_styles` here are pretty confusing.

Reply:
Ah yeah, I forgot to make that change.

![Screenshot from 2022-01-22 10-43-07](https://user-images.githubusercontent.com/69356296/150625683-90ac4b06-3591-49a3-8a3e-6d6b0184d796.png)

Reply:
Umm, can't think of a better name for `_styles`, these are the "styles" which the user explicitly passes as kwargs to overwrite the style defined by the `LogLevel`.

Reply:
I'd suggest `style_overrides`.